plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/AdditionalPayments.php

```diff
@@ -73,8 +73,11 @@ public function can_view() {
 
 		$woocommerce_payments = new WooCommercePayments();
 
-		if ( ! $woocommerce_payments->is_requested() || ( $woocommerce_payments->is_supported() && ! $woocommerce_payments->is_connected() ) ) {
-			// Hide task if WC Pay is installed via OBW, in supported country, but not connected.
+		if ( ! $woocommerce_payments->is_requested() || ! $woocommerce_payments->is_supported() || ! $woocommerce_payments->is_connected() ) {
```

Comment:
Matches condition of latest `3.3.1` patch release -> https://github.com/woocommerce/woocommerce-admin/pull/8506